Python/Salt n pepper noise removal/salt n pepper.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[2]:
+
+
+import cv2
+import numpy as np
+import matplotlib.pyplot as plt
+
+# get image
+img = cv2.imread("gaussian Noise.png", 0)
```

Comment:
no file

Reply:
Did you check my PR fully? I have already added these files.